torchreid/data/datasets/image/classification.py

```diff
@@ -239,10 +239,15 @@ def load_annotation(annot_path, data_dir, dataset_id=0):
             classes = sorted(annotation['classes'])
             class_to_idx = {classes[i]: i for i in range(len(classes))}
             images_info = annotation['images']
+            img_wo_objects = 0
             for img_info in images_info:
                 rel_image_path, img_labels = img_info
                 full_image_path = osp.join(data_dir, rel_image_path)
                 labels_idx = [class_to_idx[lbl] for lbl in img_labels if lbl in class_to_idx]
-                assert full_image_path and labels_idx
+                assert full_image_path
+                if not labels_idx:
+                    img_wo_objects += 1
```

Comment:
Did you check that an empty tuple will be processed correctly in `getitem`?

Reply:
yes, everything works as expected. 
![image](https://user-images.githubusercontent.com/68103869/131709029-6ad0cf5a-d492-4ffb-89a9-b0d9f7522daf.png)
ones are simply not written here.